openwisp_monitoring/monitoring/base/models.py

```diff
@@ -223,6 +223,8 @@ def content_type_key(self):
 
     @property
     def alert_field(self):
+        if self.field_name != self.config_dict['field_name']:
+            return self.field_name
```

Comment:
Is there a dedicated test for this condition? When I comment these lines, the test suite passes. But, I expect atleast one test to fail.

Reply:
I'll write one 👍🏻